execution/executor/src/components/in_memory_state_calculator_v2.rs

```diff
@@ -138,10 +156,13 @@ impl InMemoryStateCalculatorV2 {
             None
         };
 
+        let _timer = APTOS_EXECUTOR_OTHER_TIMERS_SECONDS
```

Comment:
I'll feel better if we anyway give it a scope

Reply:
Done